### ISSUE TITLE
False positive - Dangerous props

### ISSUE THREAD

**Issue by ThaChillera** [scottyab/rootbeer#181]
Hi,

I have a fairphone 3+ with a custom rom (lineageOS microG), but I don't have, and have never, had root. Your test app triggers on 'dangerous props', but I'm not really sure what that check does, so even though it looks like a false positive, it might be due to my custom rom!

Let me know if I can test anything for you guys

**Comment by Ktirumalsetty** [comment]
Any resolution for this i got this issue in onelus ce

**Comment by NezumiRyu** [comment]
Function: checkForDangerousProps()
Checking ro.debuggable and ro.secure

**Comment by smurfy** [comment]
Same issue with lineage 20 on oneplus 6t.  DangerousProps check triggers

**Comment by fbobraga** [comment]
Same here on Oneplus 5T with LOS 20

**Comment by gehaxelt** [comment]
Same here. What are the offending props? How can we fix this? It's preventing me from using normal apps on my deliberately root-free phone :-(

**Comment by smurfy** [comment]
My "fix" is editing `/system/build.prop` on my lineage 20 phone.

I mainly need to change `ro.debuggable` to `0`

```
adb root
adb remount
adb shell
mount -o rw,remount /
nano /system/build.prop

ro.secure=0 # should be 1
ro.debuggable=1 # should be 0
ro.build.type=userdebug" # should be "user"
```

**Comment by fbobraga** [comment]
I don't know of it's related, but antennapod.org here stopped showing notifications

Em 26 de setembro de 2023 06:00:05 BRT, Sebastian Neef ***@***.***> escreveu:

**Comment by smurfy** [comment]
> Same here. What are the offending props? How can we fix this? It's preventing me from using normal apps on my deliberately root-free phone :-(

on LOS 20 its only `ro.debuggable=1`

**Comment by KebabLord** [comment]
> My "fix" is editing `/system/build.prop` on my lineage 20 phone.
> 
> I mainly need to change `ro.debuggable` to `0` and then reboot
> 
> ```
> adb root
> adb remount
> adb shell
> mount -o rw,remount /
> nano /system/build.prop
> 
> ro.secure=0 # should be 1
> ro.debuggable=1 # should be 0
> ro.build.type=userdebug" # should be "user"
> ```

thanks for the info mate, that saved my day.

additionally in order to pass the "test keys" check, one can replace the "test-keys" with "release-keys" on build.prop

**Comment by AsbjornOlling** [comment]
This is affecting a lot of people in Denmark, since a few days ago.
The app [MobilePay](https://en.wikipedia.org/wiki/MobilePay) is the ubiquitous mobile-payments app here.I think that the vast majority of persons in Denmark use this app daily or weekly for payments.  I would estimate that MobilePay has close to 5 million active users. 

They recently released a total rewrite of the app, which now uses rootbeer. Non-standard Android devices (without root!), like ones that run LineageOS or /e/OS or similar, refuse to launch the app, because of this false positive bug. This is a big problem for a lot of people.

I have already seen a handful of people posting links to this workaround, to get mobilepay working again. I've seen people talking about this on reddit, on mastodon, in discord servers, and in matrix group chats.
I'm not sure what percentage of Android users run LineageOS or similar, but I expect that this is affecting thousands of people.

Frankly, I it's really annoying and a bit embarrassing for this project, that a 3-year old known bug has not been addressed, and is now blocking thousands of people from making payments.

**Comment by KebabLord** [comment]
@AsbjornOlling I totally agree with you, Tachi107 already created a PR fixing this issue (#192 ) but maintainer simply answered "Oz" to his PR and didn't merge it for 2 years. I hate this type of repo maintainers. They can just archive it if they'll never respond.
